drip/admin.py

```diff
@@ -46,12 +49,17 @@ def timeline(self, request, drip_id, into_past, into_future):
             into_past=int(into_past), into_future=int(into_future)+1
         ):
             shifted_drip.prune()
-            shifted_drips.append({
-                'drip': shifted_drip,
-                'qs': shifted_drip.get_queryset().exclude(id__in=seen_users)
-            })
+            shifted_drips.append(
+                {
+                    'drip': shifted_drip,
+                    'qs': shifted_drip.get_queryset().exclude(
+                        id__in=seen_users
+                    )
+                }
+            )
             seen_users.update(
-                shifted_drip.get_queryset().values_list('id', flat=True))
+                shifted_drip.get_queryset().values_list('id', flat=True)
```

Comment:
Should we add a comma at the end? @kaozdl

Reply:
I think the comma at the end only makes sense in list definitions. isn't?

Reply:
It does make sense in this case.
The trailing comma goes when you have line break inside a function's argument list.
in this case `shifted_drip... adds a line break.